Changelog.md

```diff
@@ -42,8 +44,9 @@ Compiler Features:
  * Code Generator: More efficient code for checked addition and subtraction.
  * TypeChecker: Support using library constants in initializers of other constants.
  * Yul IR Code Generation: Improved copy routines for arrays with packed storage layout.
- * Yul Optimizer: Add rule to convert ``mod(add(X, Y), A)`` into ``addmod(X, Y, A)``, if ``A`` is a power of two.
- * Yul Optimizer: Add rule to convert ``mod(mul(X, Y), A)`` into ``mulmod(X, Y, A)``, if ``A`` is a power of two.
+ * Yul Optimizer: Add rule to convert `mod(mul(X, Y), A)` into `mulmod(X, Y, A)`, if `A` is a power of two.
+ * Yul Optimizer: Add rule to convert `mod(add(X, Y), A)` into `addmod(X, Y, A)`, if `A` is a power of two.
+ * Code Generator: More efficient code for checked addition and subtraction.
```

Comment:
Looks like this reverts the change from #13357. Please remove from the PR.

Reply:
Oops. Don't know how it got in. Bad rebase.

Reply:
Fixed